### PR TITLE
feat: geocoding feedback api documentation

### PR DESCRIPTION
**Pull request by NatalieShaked**
Added documentation for Geocoding's Feedback API

### CODE REVIEW

File: docs/MapColonies/vector/Services/feedback-api/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+Geocoding's feedback api is an API that gets from the user of [Geocoding API](/docs/MapColonies/vector/Services/geocoding/README.md) the response and and puts it in redis for BI purposes.
+
+## Usage
+
+:::caution
+**You will need an API token as part of the [service authentication](/docs/MapColonies/authentication). &nbsp;**
+:::
+
+Once a user gets a response from Geocoding, the system that uses Geocoding will send the request ID and the chosen response ID back to us using the API.
```
Comment (shimoncohen) [inline docs/MapColonies/vector/Services/feedback-api/README.md:31]:
```suggestion
Once a user gets a response from `Geocoding`, the requesting system will send the `request ID` and the chosen `response ID` back to us.
```

Comment (shimoncohen) [inline docs/MapColonies/vector/Services/feedback-api/README.md:31]: Please move this to the section before `usage` and consider removing this section.

File: docs/MapColonies/vector/Services/feedback-api/README.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                "regions": [ { "region": "FRANCE", "sub_region_names": [ "Paris" ] } ]
+            }
+        }
+    ]
+}
+```
+</details>
+
+Notice this response listed 2 features.<br/><br/>
+When sending the response to the feedback api you need to fill 2 parameters:<br/>
```
Comment (shimoncohen) [inline docs/MapColonies/vector/Services/feedback-api/README.md:119]:
```suggestion
When sending the response to the `feedback API`, you need to provide two parameters:<br/>
```

File: docs/MapColonies/vector/Services/feedback-api/README.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+📍 Request ID - from the header `'x-request-id'`<br/>
+📍 Chosen Response ID - from what the user chose out of the options from the response. <br/><br/>
```
Comment (shimoncohen) [inline docs/MapColonies/vector/Services/feedback-api/README.md:121]:
```suggestion
📍 Chosen Response ID - the ID of the user's selected response (corresponding index of the result). <br/><br/>
```

File: docs/MapColonies/vector/Services/feedback-api/README.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+📍 If the user chose `"White Point Elementary School"`, the Chosen Response ID would be `0`.<br/>
+📍 If the user chose `"Wi School Paris 9"`, the Chosen Response ID would be `1`.<br/><br/>
+
+
+```curl title="Geocoding's Feedback Api Request"
+curl --location '<feedback_api_url>/feedback' \
+--header 'x-api-key: <x-api-key>' \
+--header 'x-user-id: <x-user-id>'
+```
+
```
Comment (shimoncohen) [inline docs/MapColonies/vector/Services/feedback-api/README.md:133]:
```suggestion

Then our request to the `feedback API` would look like this:</br>
```

File: docs/MapColonies/vector/Services/feedback-api/README.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+<details style={{"background-color": "#f6f8fa", border: "var(--ifm-alert-border-width) solid var(--ifm-alert-border-color)", "border-left-width": "var(--ifm-alert-border-left-width)", color: "black"}}> 
+<summary>Response 👇</summary>
+
+```json
```
Comment (shimoncohen) [inline docs/MapColonies/vector/Services/feedback-api/README.md:137]:
```suggestion
And we will receive:<br/>
```json
```

**Review by shimoncohen — CHANGES_REQUESTED** [review]

File: docs/MapColonies/vector/Services/feedback-api/README.md
[the remark this answers is not in the document]
Comment (NatalieShaked) [reply docs/MapColonies/vector/Services/feedback-api/README.md:1]: You said you'll do this.

File: docs/MapColonies/vector/Services/feedback-api/README.md (hunk 1 above)
Comment (NatalieShaked) [reply docs/MapColonies/vector/Services/feedback-api/README.md:31]: You mean just remove the usage title?

**Comment by NivGreenstein** [comment]
I would add `caution` admonition that systems that won't comply will be blocked from using Geocoding service. 
@shimoncohen you're opinion?

File: docs/MapColonies/vector/Services/feedback-api/README.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+                "names": { "en": [ "Wi School Paris 9" ], "fr": [ "Ecole Wi Paris 9" ], "default": [ "Wi School Paris 9" ], "display": "Wi School Paris 9" },
+                "placetype": "education",
+                "sub_placetype": "school",
+                "regions": [ { "region": "FRANCE", "sub_region_names": [ "Paris" ] } ]
+            }
+        }
+    ]
+}
+```
+</details>
```
Comment (NivGreenstein) [inline docs/MapColonies/vector/Services/feedback-api/README.md:116]: Make sure to be clear that part of the response is `x-request-id` header. It is not clear here.

File: docs/MapColonies/vector/Services/feedback-api/README.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+📍 If the user chose `"Wi School Paris 9"`, the Chosen Response ID would be `1`.<br/><br/>
+
+Then our request to the `feedback API` would look like this: <br/>
+
+```curl title="Geocoding's Feedback Api Request"
+curl --location '<feedback_api_url>/feedback' \
+--header 'x-api-key: <x-api-key>' \
+--header 'x-user-id: <x-user-id>'
+```
+
```
Comment (NivGreenstein) [inline docs/MapColonies/vector/Services/feedback-api/README.md:134]: This is not the right curl. 
First, it isn't POST request. missing body payload

File: docs/MapColonies/vector/Services/feedback-api/README.md (hunk 6 above)
Comment (NatalieShaked) [reply docs/MapColonies/vector/Services/feedback-api/README.md:116]: Did it, want to make sure it's approved by Shimon

File: docs/MapColonies/vector/Services/feedback-api/README.md [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+Lets think of the current response as the actual response in this case:<br/>
+📍 If the user chose `"White Point Elementary School"`, the Chosen Response ID would be `0`.<br/>
+📍 If the user chose `"Wi School Paris 9"`, the Chosen Response ID would be `1`.<br/><br/>
+
+Then our request to the `feedback API` would look like this: <br/>
+
+```curl title="Geocoding's Feedback Api Request"
+curl --location --request POST '<feedback_api_url>/feedback' \
+--header 'x-api-key: <x-api-key>' \
+--header 'x-user-id: <x-user-id>' \
```
Comment (NivGreenstein) [inline docs/MapColonies/vector/Services/feedback-api/README.md:138]: We do not require `x-user-id` in Feedback API request

**Review by NivGreenstein — APPROVED** [review]
LGTM!

**Review by NivGreenstein — APPROVED** [review]
fixed comments! ready to be marged

**Review by shimoncohen — APPROVED** [review]

**Pull request merged**